girder/api/rest.py

```diff
@@ -18,6 +18,7 @@
 ###############################################################################
 
 import cherrypy
+import cherrypy.lib
```

Comment:
Yes, we do need the explicit import. By design, CherryPy's root __init__.py file [1] does not do an "import lib" anywhere, so it's not available in the top "cherrypy" namespace.

Also, note that CherryPy's careful use of modules and imports presents a very clean public namespace. This is something Girder should eventually start to consider.

[1] https://github.com/cherrypy/cherrypy/blob/3.3.0/cherrypy/__init__.py